### DISCUSSION TITLE
The size of Bulk Crap Uninstaller

### DISCUSSION THREAD

**Discussion by ags1234** [Klocman/Bulk-Crap-Uninstaller#667] [General]
I just tried BCUninstaller and compared it with others.
There is an issue that for me is hard to get over: the size.

CrystalIDEA Uninstall Tool ~ 10 MiB
HiBit Uninstaller ~ 14 MiB
Soft Organizer ~ 37 MiB
Revo Uninstaller ~ 52 MiB
BCUninstaller_5.8.1_portable ~ 363 MiB  <-- (?!)

In fact BCUninstaller might be as big as all the other well known uninstallers available today for Windows combined!

Bulk - the software that helps get rid of bulk is itself very bulky
Crap - depends of .net framework Microsoft crap
Uninstaller - indeed, it definitely is one

So taking into account the program name, its size, and its purpose, something does not add up there.
Just to be clear, this is not only a problem of used bandwidth and disc space. It is a problem of efficiency.

**Comment by Klocman** [comment]
You can use the net6.0 version which is only 11MB, it's next to other builds in releases.
